### PR TITLE
Adds support for many menu providers, Adds Zielone Tarasy 

### PR DESCRIPTION
**Pull request by pawbuj**

### CODE REVIEW

File: src/Menu/Menu.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     */
+    public function getItems(): array
+    {
+        return $this->items;
+    }
+
+    /**
+     * @return string
+     */
+    public function getRestaurant()
```
Comment (adamwojs) [inline src/Menu/Menu.php]:
```suggestion
    public function getRestaurant(): string
```

File: src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    /**
+     * @return array
+     */
+    private function parse(): array
+    {
+        $children = $this
+            ->crawler
+            ->filter('#comp-josrqp2g')
```
Comment (adamwojs) [inline src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php]: Please move this selector to const 😉

File: src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    {
+        return 'Zielone Tarasy Restro Bar Gallery';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isMenuAvailable(DateTimeInterface $date): bool
+    {
+        // todo
```
Comment (adamwojs) [inline src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php]: Please resolve or remove `todo` comment

File: src/Menu/Menu.php (hunk 1 above)
Comment (pawbuj) [reply src/Menu/Menu.php]: done

File: src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php (hunk 2 above)
Comment (pawbuj) [reply src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php]: done

File: src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php (hunk 3 above)
Comment (pawbuj) [reply src/Menu/Provider/ZieloneTarasy/ZieloneTarasyMenuProvider.php]: done

**Comment by adamwojs** [comment]
Thank you for contribution @pawbuj! :beers:

**Pull request merged**